### ISSUE TITLE
Don't use the UI from the server

### ISSUE THREAD

**Issue by tommie-lie** [eaas-framework/emil-admin-ui#1]
This issue is about the clientside-rendering branch!

I recently redesigned the way we access the UI to get rid of JSF and JSP.
This means that it's not necessary to render an iframe from the EaaS server. The "control URL" now is a base URL to a client API. The guacamole tunnel can be reached at $controlURL/tunnel and the current state of the component can be queried at $controlURL/state.
Please refer to the client-iframe.html file in the clientside-rendering branch to see it in action.
I tried to wrap my mind around AngularJS and tried to figure out how I can get the control URL from the controller into a <script> tag in the view but I couldn't find any recommended way ($scope seems to work but this is frowned upon?). That's the sole reason why there is still an iframe. If we could access the control URL from within the JavaScript, we could do the rendering completely without the iframe.

Please try to fix my mess and see if you can integrate the code from client-iframe.html into partials/wf-s/emulator.html (or any other suitable place). Ideally, there would be no iframe at all, just a div that is shown as soon as the connection to the emulator is ready.

Once this is done, it obviously has to be merged into emil-ui as well.

Our next goal in the UI department is to encapsulate what is currently in the client-iframe.html and eaas/ directory into a "eaas javascript library" so that users who create their own UI can just say `new EaasClient("#id-of-div-tag")` and the rest is done for them.

**Comment by nickrussler** [comment]
I pushed https://github.com/eaas-framework/emil-admin-ui/commit/056fe84270e53a52291feab3c97e34aa7c6491ca which adds an exemplary call from the emulator controller to the on-page emulator script.

**Comment by tommie-lie** [comment]
I started a small library in `eaas/*` and everything seems to work now. Closing.
